nilearn/connectome/tests/test_connectivity_matrices.py

```diff
@@ -421,3 +483,93 @@ def test_connectivity_measure_outputs():
                     d = np.sqrt(np.diag(np.diag(prec)))
                     assert_array_almost_equal(d.dot(cov_new).dot(d), -prec +
                                               2 * np.diag(np.diag(prec)))
+
+    # Check the mean_
+    for kind in kinds:
+        conn_measure = ConnectivityMeasure(kind=kind)
+        conn_measure.fit_transform(signals)
+        assert_equal((conn_measure.mean_).shape, (n_features, n_features))
+
+    # Check vectorization option
+    for kind in kinds:
+        conn_measure = ConnectivityMeasure(kind=kind)
+        connectivities = conn_measure.fit_transform(signals)
+        conn_measure = ConnectivityMeasure(vectorize=True, kind=kind)
+        vectorized_connectivities = conn_measure.fit_transform(signals)
+        assert_array_almost_equal(vectorized_connectivities,
+                                  sym_matrix_to_vec(connectivities))
+
+    # Check not fitted error
+    assert_raises_regex(
+        ValueError, 'has not been fitted. ',
+        ConnectivityMeasure().inverse_transform,
+        vectorized_connectivities)
+
+    # Check inverse transformation
+    kinds.remove('tangent')
+    for kind in kinds:
+        # without vectorization
+        conn_measure = ConnectivityMeasure(kind=kind)
+        connectivities = conn_measure.fit_transform(signals)
+        assert_array_almost_equal(
+            conn_measure.inverse_transform(connectivities), connectivities)
+
+        # with vectorization
+        # if diagonal has not been discarded
+        conn_measure = ConnectivityMeasure(kind=kind, vectorize=True)
```

Comment:
do you think these blocks are necessary ? Basically same blocks are repeated below, I guess. Is it not ?